client/app/components/dynamic-table/index.js

```diff
@@ -12,8 +12,8 @@ function DynamicTable($sanitize) {
   this.orderByReverse = false;
 
   this.pageChanged = () => {
-    const first = this.count * (this.page - 1);
-    const last = this.count * (this.page);
+    const first = this.itemsPerPage * (this.page - 1);
+    const last = this.itemsPerPage * (this.page);
```

Comment:
But we do assign a value to `this.count` in the component initialization, so even if the parent doesn't provide a value we give it a default.

Also I couldn't reproduce this behavior (ran `select 1` and saw results). Maybe there is a different issue here?